### PR TITLE
Add real URL for student timesheets page (#723)

### PR DESCRIPTION
**Pull request by NathanielRudenberg**
* Enables the student timesheets page for the public
* Adds a link to the timesheets page in the Quick Links dialog

image

### CODE REVIEW

File: src/views/Timesheets/components/SavedShiftsList/index.js [hunk 1]
```diff
@@ -83,6 +84,21 @@ export default class SavedShiftsList extends Component {
     });
   }
 
+  componentDidMount() {
```
Comment (aridospassos) [inline src/views/Timesheets/components/SavedShiftsList/index.js:87]: This componentDidMount is doing a lot of stuff. Might be worth adding a little descriptive comment to help us out when we pick up working on it.

File: src/views/Timesheets/components/SavedShiftsList/index.js [hunk 2]
```diff
@@ -83,6 +84,21 @@ export default class SavedShiftsList extends Component {
...
+    } else {
+      shouldGetSupervisors = this.props.cardTitle === "Saved Shifts" && supervisorIdsReady && (!this.state.directSupervisor || !this.state.reportingSupervisor) && shifts !== null;
+    }
+    if (shouldGetSupervisors) {
+      this.getSupervisors();
+    }
+  }
+
   componentDidUpdate() {
     let {shifts, directSupervisor, reportingSupervisor} = this.props;
```
Comment (aridospassos) [inline src/views/Timesheets/components/SavedShiftsList/index.js:103]: I think adding a little description of what exactly is happening here would be good.

**Comment by NathanielRudenberg** [comment]
I agree with these needs. I'll spend the next day or so making the required changes and then I'll push them to this branch.

My one concern is the process for getting the new changes back into `develop`.  If I were to make the changes and merge those to develop before making a pull request into master, that might potentially bring in unfinished practicum work as well. Pushing them here instead and then merging to master will mean that master has changes that aren't in develop.

**Comment by jahnueldorelus** [comment]
This issue been solved in the Pull Request "PWA to Develop"

**Pull request closed without merging**